scripts/validate_docstrings.py

```diff
@@ -946,9 +977,14 @@ def header(title, width=80, char='#'):
                            action='store_true', help='if this flag is set, '
                            'deprecated objects are ignored when validating '
                            'all docstrings')
+    argparser.add_argument('--ignore_known_fail', default=False,
```

Comment:
Thoughts on the default? I would expect that if you're whitelisting thees failures, you wouldn't need to also pass `--ignore-known-fail`. Not a big deal either way since this is just in CI.

Can you add a couple sentences to `contributing_docstring.rst` saying when and how to use this?